src/main/java/duke/logic/task/Task.java

```diff
@@ -0,0 +1,75 @@
+package duke.logic.task;
+
+/**
+ * Task parent class.
+ */
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    /**
+     * Constructor for Task object.
+     *
+     * @param description Description of Task.
+     */
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    /**
+     * Constructor for Task object with boolean.
+     *
+     * @param description Description of task.
+     * @param isDone Whether task is marked complete.
+     */
+    public Task(String description, Boolean isDone) {
+        this.description = description;
+        this.isDone = isDone;
+    }
+
+    public boolean getIsDone() {
```

Comment:
It might be better to name this `isDone()` instead to make it sound more like a boolean method! It also allows linters to automatically verify that this style rule is being followed.